health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/MultiSelectDropdown.js

```
@@ -0,0 +1,1010 @@
+    setDummySelected(res);
+  }
+
+  const handleClearAll = () => {
+    setDummySelected([]);
+  };
+  return (
+    <PopUp
+      className={"selecting-boundaries-pop"}
+      type={"default"}
+      heading={`${t((hierarchyType + "_" + boundaryType).toUpperCase())} ${t("DIGIT_SELECT")}`}
+      children={[]}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)